src/scan.js

```diff
@@ -66,7 +72,14 @@ function getInstanceInfo({ node, filePath, importInfo }) {
     if (attribute.type === "JSXAttribute") {
       const { name, value } = attribute;
       const propName = name.name;
-      const propValue = getPropValue(value);
+      const propValue = customGetPropValue
+        ? customGetPropValue({
+            node: value,
+            propName,
+            componentName,
+            defaultGetPropValue: getPropValue,
```

Comment:
One additional param I added here was the `defaultGetPropValue`, so that consumers can use the default handling where they don't care to specify a specific `propName` or `componentName`